modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift-sdn/configuring-egress-firewall.adoc
+
+[id="nw-egressnetworkpolicy-about_{context}"]
+= How an egress firewall works in a project
+
+As an {product-title} cluster administrator, you can use an _egress firewall_ to
+limit the external IP addresses that some or all Pods can access from within the
+cluster. Egress firewall policy supports the following scenarios:
+
+- A Pod can only connect to internal hosts, and cannot initiate connections to
+the public Internet.
+- A Pod can only connect to the public Internet, and cannot initiate connections
+to internal hosts that are outside the {product-title} cluster.
+- A Pod cannot reach specified internal subnets or hosts outside the {product-title} cluster that should be unreachable.
+
+An egress firewall policy is configured by creating an EgressNetworkPolicy Custom Resource (CR) object and specifying an IP address range in CIDR format or by specifying a DNS name.
+For example, you can allow one project access to a specified IP range but deny the same access to a different project, or you can restrict application developers from updating from Python pip mirrors, and force updates to come only from approved sources.
+
+[IMPORTANT]
+====
+You must have OpenShift SDN configured to use either the network policy or multitenant modes to configure egress firewall policy.
+
+If you are using network policy mode, egress policy is compatible with only one policy per project, and will not work with projects that share a network, such as global projects.
```

Comment:
"only one policy per project" is that egress policy or network policy?  I assume egress.

Reply:
Tested both multitenet and network-policy mode in 4.3.0-rc.3, more than one egress policies can be created per project.

Reply:
@weliang1 you can *create* more than one EgressNetworkPolicy, but if you do, the expected behavior is that the sdn pods will log an error and all egress traffic from that namespace will be dropped. There ought to be QE tests confirming this.
(We maybe ought to think about having an admission controller so we can actually stop people from creating the conflicting policies...)

Reply:
(ie, to clarify, the docs should say you can only create one ENP per namespace. We don't enforce that at object-creation time, so people can screw up their cluster by creating more than one.)